pandas/tests/series/indexing/test_indexing.py

```diff
@@ -292,9 +299,9 @@ def test_underlying_data_conversion(using_copy_on_write):
 
 
 def test_preserve_refs(datetime_series):
-    seq = datetime_series[[5, 10, 15]]
-    seq[1] = np.NaN
-    assert not np.isnan(datetime_series[10])
+    seq = datetime_series.iloc[[5, 10, 15]]
+    seq.iloc[1] = np.NaN
```

Comment:
same here

Reply:
I didn't modify this one either - I presume you're commenting on the merge commit?